src/routes/(settings)/settings/billing/success/unlimited/+page.svelte

```
@@ -0,0 +1,113 @@
+<script lang="ts">
+  import { Button } from "$lib/components/ui/button";
+  import { CheckCircle, Zap, ArrowRight, Infinity } from "@lucide/svelte";
```

ESLint

[error] 'Zap' is defined but never used.

(@typescript-eslint/no-unused-vars)